Lib/calendar.py

```diff
@@ -132,17 +132,10 @@ def _monthlen(year, month):
 
 
 def _prevmonth(year, month):
-    if month == 1:
-        return year-1, 12
-    else:
-        return year, month-1
-
+    return [year-1, 12] if month == 1 else [year, month-1]
```

Comment:
One of the reasons we don't usually accept changes like this is because they might cause subtle changes. For example, you've switched the return type from a tuple to a list. I haven't checked if that will cause an actual problem or not in this case.